### ISSUE TITLE
Firmware 1.4 and IOS version 7.01 - App won't connect

### ISSUE THREAD

**Issue by JerryBlack** [dride/dride-app-issues#69]
@saoron - I note that @BillieTheKid is running 7.1 on Android. I don't see that on the iPhone yet. 
As noted on the dride.io forum, **I'm about to try reformatting the SD card,** but it's interesting to note that (after reinstalling the firmware 1.4 with no change in behaviour), the dride is still working, it is providing hotspot, I can connect to it with my MacBookPro and download a movie to confirm that it is functioning (with GPS location). I just can't connect to it with the app on IOS version 7.01 Oct 4 2019, which appears to be the latest version for IOS?

**Comment by JerryBlack** [comment]
@saoron After formatting the SD card (exFAT), the dride zero continues to function, as determined by connection via the MacBookPro, e.g. hot spot, videos recorded, date/time, speed OK, but still no connectivity using the app IOS version 7.01. 
Cheers

**Comment by MyDefender** [comment]
I had already updated to FW 1.4 from GitHub which resulted in not being able to connect to the app iOS 7.1. I then used iOS 7.11 and followed the instructions in https://dride.io/forum/-fix-for-firmware-14-____Y6OFQaNmbmlgtTsXwdOx, download V.1.4.1, put that on the SD card but it would not install - probably because it is named 'FW96663A-1.4.1.bin' and not 'FW96663A.bin'. I ended up installing V1 of the FW and was then able to update to 1.4 via the latest version of the app. I now seem to have GPS data on the recording for the first time ever! Let's hope that is not a passing phase :)
